internal/server/statvar_hierarchy_util.go

```diff
@@ -123,16 +125,40 @@ func (index *SearchIndex) update(
 	}
 }
 
+// Helper to build a set of ignored SVGs.
+func getIgnoredSVGHelper(
+	ignoredSvg map[string]string,
+	rawSvg map[string]*pb.StatVarGroupNode,
+	svgID string) {
+	ignoredSvg[svgID] = ""
+	svgData, ok := rawSvg[svgID]
+	if ok {
```

Comment:
nit: can write in one line as:

if svgData, ok := rawSvg[svgID]; ok {
...
}

Reply:
done